From 5bdc3c6732276886b633d26d032958d1fd1e4b84 Mon Sep 17 00:00:00 2001
From: James Tirta Halim <tirtajames45@gmail.com>
Date: Wed, 21 Jan 2026 23:35:00 +0700
Subject: [PATCH] disable: nvidia

---
 Makefile         |  6 +++---
 src/config.def.h | 10 +++++-----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/Makefile b/Makefile
index fc2fe2c..4472f44 100644
--- a/Makefile
+++ b/Makefile
@@ -30,9 +30,9 @@ X11FLAGS += -lX11
 # Alsa (comment to disable)
 ALSAFLAGS += -lasound
 
-# NVML (comment to disable)
-NVMLLIB = /opt/cuda/lib64
-NVMLFLAGS += -L$(NVMLLIB) -lnvidia-ml
+# # NVML (comment to disable)
+# NVMLLIB = /opt/cuda/lib64
+# NVMLFLAGS += -L$(NVMLLIB) -lnvidia-ml
 
 # # FreeBSD (uncomment)
 # FREEBSDFLAGS += -L/usr/local/lib -I/usr/local/include
diff --git a/src/config.def.h b/src/config.def.h
index d0dd15f..03ddc79 100644
--- a/src/config.def.h
+++ b/src/config.def.h
@@ -22,15 +22,15 @@
 /* Use unlocked stdio functions, since single-threaded. */
 #	define USE_UNLOCKED_IO 1
 
-/* Monitor Nvidia GPU. nvidia-settings as fallback. Comment to disable. */
-#	define USE_NVIDIA 1
+/* /1* Monitor Nvidia GPU. nvidia-settings as fallback. Comment to disable. *1/ */
+/* #	define USE_NVIDIA 1 */
 
 /* Monitor audio volume, requires ALSA. Comment to disable. */
 #	define USE_ALSA 1
 
-/* Monitor Nvidia GPU, requires CUDA. Comment to disable. */
-#	define USE_NVML    1
-#	define NVML_HEADER "/opt/cuda/targets/x86_64-linux/include/nvml.h"
+/* /1* Monitor Nvidia GPU, requires CUDA. Comment to disable. *1/ */
+/* #	define USE_NVML    1 */
+/* #	define NVML_HEADER "/opt/cuda/targets/x86_64-linux/include/nvml.h" */
 
 /* Path to CPU temperature */
 #	define CPU_TEMP_FILE "/sys/class/hwmon/hwmon3/temp2_input"
-- 
2.52.0

