From 7082779870e5915be1e475a2f619503fc65354a2 Mon Sep 17 00:00:00 2001
From: James Tirta Halim <tirtajames45@gmail.com>
Date: Wed, 21 Jan 2026 22:58:00 +0700
Subject: [PATCH] nvml: disable

---
 Makefile         | 4 ++--
 src/config.def.h | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index fc2fe2c..8a22c4e 100644
--- a/Makefile
+++ b/Makefile
@@ -31,8 +31,8 @@ X11FLAGS += -lX11
 ALSAFLAGS += -lasound
 
 # NVML (comment to disable)
-NVMLLIB = /opt/cuda/lib64
-NVMLFLAGS += -L$(NVMLLIB) -lnvidia-ml
+# NVMLLIB = /opt/cuda/lib64
+# NVMLFLAGS += -L$(NVMLLIB) -lnvidia-ml
 
 # # FreeBSD (uncomment)
 # FREEBSDFLAGS += -L/usr/local/lib -I/usr/local/include
diff --git a/src/config.def.h b/src/config.def.h
index d0dd15f..8842c5b 100644
--- a/src/config.def.h
+++ b/src/config.def.h
@@ -29,8 +29,8 @@
 #	define USE_ALSA 1
 
 /* Monitor Nvidia GPU, requires CUDA. Comment to disable. */
-#	define USE_NVML    1
-#	define NVML_HEADER "/opt/cuda/targets/x86_64-linux/include/nvml.h"
+/* #	define USE_NVML    1 */
+/* #	define NVML_HEADER "/opt/cuda/targets/x86_64-linux/include/nvml.h" */
 
 /* Path to CPU temperature */
 #	define CPU_TEMP_FILE "/sys/class/hwmon/hwmon3/temp2_input"
-- 
2.52.0

